Displaying TS code examples in the docs (#10966)
handsontable/handsontable: docs/README.md

* `npm run docs:scripts:link-assets` – Prepares the `next` documentation version's CSS and JavaScript.
* `npm run docs:review [COMMIT_HASH]` – Deploys the documentation locally at a `[COMMIT_HASH]` commit.
* `npm run docs:test:example-checker` – Runs the tests that checks if all Docs examples work.

> * `npm run docs:scripts:generate-js-examples` – Generate js examples from ts files (need to run this script before first `docs:start` and every time after change ts file content in `docs/content/guides`)

Author: ```suggestion
* `npm run docs:scripts:generate-js-examples` – Generate js version of the code examples (need to run this script before first `docs:start` and every time some `*.ts` inside `docs/content/guides` changes)
```

> * `npm run docs:scripts:generate-js-examples` – Generate js examples from ts files (need to run this script before first `docs:start` and every time after change ts file content in `docs/content/guides`)

Author: Please, add `docs:scripts:generate-js-examples` as a step in `Getting started with Handsontable documentation` section of the readme.

Reviewer: added
